src/crystalla/matrix.cr

```diff
@@ -216,6 +241,27 @@ module Crystalla
       Matrix.rows rows
     end
 
+    class SVD
```

Comment:
Maybe move this class to its own file, so we don't continue bloating matrix.cr? WDYT?